lib/createForm.js

```diff
@@ -35,7 +35,11 @@ export async function createEmptyForm(bestuurseenheid) {
         mu:uuid """${publicServiceId}""" ;
         adms:status <http://lblod.data.gift/concepts/79a52da4-f491-4e2f-9374-89a13cde8ecd> ;
         ${spatialsPreparedStatement.length ? spatialsPreparedStatement : 'dct:spatial ?spatial;'}
-        pav:createdBy ${sparqlEscapeUri(bestuurseenheid)}.
+        pav:createdBy ${sparqlEscapeUri(bestuurseenheid)};
+        lpdcExt:competentAuthorityLevel <https://productencatalogus.data.vlaanderen.be/id/concept/BevoegdBestuursniveau/Lokaal>;
```

Comment:
Hardcoded to "Lokaal" for now. Not sure if this is good enough or that we need to map the "bestuurseenheid" to the correct level somehow.

Reply:
We're only going to prefill the authorities so we should remove the hardcoded level values.